helm/hpcc/values.schema.json

```diff
@@ -569,6 +569,33 @@
       },
       "additionalProperties": false
     },
+    "pageCache": {
+      "description": "Details of the page cache ",
+      "type": "object",
+      "properties": {
+        "sizeMB": {
+          "description": "How much disk space (in MB) to reserve for the disk cache",
+          "type": "integer"
+        },
+        "pageSize": {
+          "description": "How many bytes does each page occupy in the cache - must be a power of 2 and a multiple of the index node size",
+          "type": "integer"
+        },
+        "readSize": {
+          "description": "Recommended size when reading from the cache",
```

Comment:
should be a power of 2 and >= pageSize ?

Reply:
actually, should be a factor of pageSize.  <= pageSize.  This is the size when reading data from the cache.